tools/ddsrouter_tool/test/compose/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.16.3)
+
+project(ddsrouter_tool_compose_tests LANGUAGES CXX)
+
+enable_testing()
+include(CTest)
+
+# Name of files to test
+set(TESTS
+    repeater
+)
+
+foreach(TEST ${TESTS})
+    file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/${TEST} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+endforeach()
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/docker-compose.sh
```

Comment:
It's preferred to use `configure_file` as this will track the changes in the file and re-run CMake to re-configure the file and generate the build system again.